### PR TITLE
[PowerToys] Add Environment Variables docs

### PR DESCRIPTION
**Pull request by stefansjfw**

### CODE REVIEW

**Comment by stefansjfw** [comment]
To be merged only with the PowerToys 0.75 release if Environment Variables ships in it.

File: hub/powertoys/index.md [hunk 1]
```diff
@@ -69,6 +69,17 @@ The currently available utilities include:
...
 :::row-end:::
 
+### Environment Variables
+
+:::row:::
+    :::column:::
+        [![Environment Variables screenshot.](../images/powertoys-environment-variables.png)](environment-variables.md)
+    :::column-end:::
+    :::column span="2":::
+        [Environment Variables](environment-variables.md) app offers a nice and easy way to manage environment variables.Also, it allows you to create profiles to be able to manage set of variables at once.
```
Comment (alvinashcraft) [inline hub/powertoys/index.md]:
```suggestion
        [Environment Variables](environment-variables.md) offers an easy and convenient way to manage environment variables. It also allows you to create profiles for managing a set of variables together.
```

File: hub/powertoys/environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+title: PowerToys Environment Variables for Windows
+description: Environment Variables is a convenient way to manage environment variables.
+ms.date: 10/24/2023
+ms.topic: article
+no-loc: [PowerToys, Windows, Environment Variables, Win]
+---
+
+# Environment Variables
+
+Environment Variables app offers a nice and easy way to manage environment variables. Also, you can create profiles and manage multiple variables at once. Profile variables have precedence over User and System variables. In the background, applying the profile adds variables to User environment variables. When profile is applied, if there is an existing User variable with the same name, backup variable is created in User variables which will be reverted to original one on profile un-apply.
```
Comment (alvinashcraft) [inline hub/powertoys/environment-variables.md]:
```suggestion
Environment Variables offers an easy and convenient way to manage environment variables. It also allows you to create profiles for managing a set of variables together. Profile variables have precedence over User and System variables. Applying the profile adds variables to User environment variables in the background. When a profile is applied, if there is an existing User variable with the same name, a backup variable is created in User variables which will be reverted to the original value on profile un-apply.
```

File: hub/powertoys/environment-variables.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Applied variables list shows the current state of the environment, respecting the order of evaluation of environment variables (Profile -> User -> System). Evaluated Path variable value is shown at the top of the list. 
+
+![PowerToys Environment Variables screenshot.](../images/powertoys-environment-variables.png)
+
+## Edit/Remove variable
+
+To edit or remove a variable (profile, User or System) for specific variable select more options button (**•••**):
```
Comment (alvinashcraft) [inline hub/powertoys/environment-variables.md]:
```suggestion
To edit or remove a variable (profile, User or System) for a specific variable, select the more options button (**•••**):
```

Using "variable" twice in the sentence doesn't feel correct to me. Could the sentence be changed to start with "To edit or remove a value"?

File: hub/powertoys/environment-variables.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+![PowerToys Environment Variables: Edit/Remove variable](../images/powertoys-environment-variables-edit-variable.gif)
+
+## Add profile
+
+To add new profile:
```
Comment (alvinashcraft) [inline hub/powertoys/environment-variables.md]:
```suggestion
To add a new profile:
```

File: hub/powertoys/environment-variables.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+ - Select **New profile**
+ - Enter profile name
+ - Set Enabled toggle to On to apply the profile right after creaton
+ - Select **Add variable** to add variables to profile (either new variable or existing User/System variables).
+ - Select **Save**
+
+![PowerToys Environment Variables: Add profile](../images/powertoys-environment-variables-add-profile.gif)
+
+To edit or remove the profile, select profile more options button (**•••**).
```
Comment (alvinashcraft) [inline hub/powertoys/environment-variables.md]:
```suggestion
To edit or remove a profile, select the more options button (**•••**) on the desired profile.
```

File: hub/powertoys/environment-variables.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+## Apply profile
+
+To apply profile, set profile toggle to On. Only one profile can be applied at the time. Applied variables list will show applied profile variables at the top (below Path variable):
```
Comment (alvinashcraft) [inline hub/powertoys/environment-variables.md]:
```suggestion
To apply a profile, set the profile toggle to On. Only one profile can be applied at a time. The Applied variables list will show applied profile variables at the top (below Path variable):
```

File: hub/powertoys/environment-variables.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+![PowerToys Environment Variables: Apply profile](../images/powertoys-environment-variables-apply-profile.gif)
+
+## Settings
+
+From the Settings menu, the following options can be configured:
+
+| Setting | Description |
+| :--- | :--- |
+| Launch as administrator | Launch as administrator to be able to manage System variables. If disabled, only profile and User variables can be modified. Environment Variables is started as administrator by default. |
```
Comment (alvinashcraft) [inline hub/powertoys/environment-variables.md]:
```suggestion
| Launch as administrator | Launch as administrator allow management of System variables. If disabled, only profile and User variables can be modified. Environment Variables is started as administrator by default. |
```

**Review by alvinashcraft — COMMENTED** [review]
I left some suggestions inline. You should also modify the `dev-environment/toc.yml` file to add this new page to the table of contents in the docs.

File: hub/powertoys/environment-variables.md (hunk 3 above)
Comment (stefansjfw) [reply hub/powertoys/environment-variables.md]: Changed to `To edit or remove a variable (profile, User or System), select the more options button (**•••**) on the desired variable:` to align with profile on line 35

Comment (stefansjfw) [reply hub/powertoys/environment-variables.md]: does that sound better?

**Comment by stefansjfw** [comment]
@alvinashcraft Thanks for the review! I committed suggestions

**Review by alvinashcraft — APPROVED** [review]
Looks good. I'll merge this when the next release goes out. Thanks!

**Pull request merged**